doc/stdlib/dates.rst

```diff
@@ -535,6 +535,31 @@ Periods
 
    Construct a ``Period`` type with the given ``v`` value. Input must be losslessly convertible to an ``Int64``\ .
 
+.. function:: CompoundPeriod(periods) -> CompoundPeriod
+
+   .. Docstring generated from Julia source
+
+   Construct a ``CompoundPeriod`` from a ``Vector`` of ``Period``\ s. The provided ``Period``\ s will be simplified using the following rules:
+
+   * All ``Period``\ s of the same type will be added together
```

Comment:
Also:
> Because of this canonicalization, `CompoundPeriod` is also useful to convert time periods into more human-recognizable forms.  For example, `Dates.CompoundPeriod(Dates.Minute(50000))` yields 4 weeks, 6 days, 17 hours, and 20 minutes.